### ISSUE TITLE
unpack: doesn't preserve mtime

### ISSUE THREAD

**Issue by cyphar** [cyphar/umoci#1]
This is because the OCI tooling doesn't appear to handle this properly (`image.CreateRuntimeLayoutBundle` is to blame). It will probably be a good idea to from-scratch implement this feature inside `umoci` so that there is more than one implementation of this unpacking functionality.

```
% umoci unpack --image opensuse --ref latest --bundle bundle1
% sleep 30s
% umoci unpack --image opensuse --ref latest --bundle bundle2
% gomtree -f bundle1/*.mtree -p bundle2/rootfs >/dev/null
% echo $?
1
```

Where all of the `gomtree` failed keywords are related to time.

**Comment by cyphar** [comment]
Opened a bug report upstream. opencontainers/image-tools#74

**Comment by cyphar** [comment]
This has been fixed in 7654ebc76ec62c39f48eac45183ec2766132db33. In addition, I dealt with the issue of `.` not having entries in most images (we just use the Unix epoch as the `atime` and `mtime` to make sure that bundles are completely consistent).
